examples/angular2/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
```

Comment:
We can't require every app to do it and this one don't. Also it's explicit. https://github.com/tastejs/todomvc/pull/1182#issuecomment-76743705